ansible_collections/arista/avd/roles/eos_designs/templates/network_services/vlan-interfaces.j2

```diff
@@ -75,6 +75,25 @@ vlan_interfaces:
         vrf: {{ svi_ip_helpers[helper_ip].source_vrf | arista.avd.default }}
 {%                 endfor %}
 {%             endif %}
+{%             if svi.ospf.enabled is arista.avd.defined(true) %}
```

Comment:
add a check if `vrf.ospf.enabled is arista.avd.defined(true)`

Reply:
Fixed.